src/main/java/com/bynder/sdk/query/upload/UploadQuery.java

```diff
@@ -64,12 +71,17 @@ public UploadQuery setAudit(final Boolean audit) {
         return this;
     }
 
-    public String getMetaproperty() {
-        return metaproperty;
+    public List<MetapropertyAttribute> getMetaproperties() {
+        return metaproperties;
     }
 
-    public UploadQuery setMetaproperty(String metaproperty) {
-        this.metaproperty = metaproperty;
+    public UploadQuery setMetaproperty(String metapropertyUuid, String optionName) {
```

Comment:
Is this (intented to be) used somewhere? And wouldn't it make more sense (in terms of simplicity and consistency) to have a `setMetaproperties()` like in `SaveMediaQuery`?

Reply:
I intend to use it. I think it is a practical way to set an individual property rather than require working with a new List every time you need to change them.

Reply:
Allright I'm fine with it, but I'd at least call it `addMetaproperty` to avoid any confusion it's one of the setters, but rather adding data to the list of metaprops.

Reply:
That's an excellent idea. Just changed it.